### DISCUSSION TITLE
compress_json() not working?

### DISCUSSION THREAD

**Discussion by freddysongg** [microsoft/LLMLingua#124] [Q&A] [answered]
Hey all, 

I was beginning to use LLMLingua and I tried to configure it so that it can optimally compress my JSON data but when I ran it, I get an attribute error:
AttributeError: 'PromptCompressor' object has no attribute 'compress_json'

I'm following the documentation by using 

llm_lingua = PromptCompressor(device_map="mps")
(I'm on MAC)

Is there something that I'm missing?

**Comment by SiyunZhao** [comment, marked as answer]
Hi @MrFrooty , thank you for your support.
The reason for this issue might be that this feature has not been released yet, so the current version you are using has not updated compress_json method. If you want to use it, you can update to the latest github version by using `pip install git+ https://github.com/microsoft/LLMLingua.git`
.
